kore/src/Kore/Exec/GraphTraversal.hs

```diff
@@ -397,7 +411,7 @@ data StepResult a
 data TraversalResult a
     = -- | remaining queue length and stuck results (always at most
       -- maxCounterExamples many).
-      GotStuck Int [a]
+      GotStuck Int [Either a a]
```

Comment:
Should maybe modify the comment here to explain what `Left` and `Right` in `GotStuck` mean...